test/helpers/user_blocks_helper_test.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+require "test_helper"
+include ApplicationHelper
```

Comment:
The other question is rubocop is complaining about the use of html_safe I assume, given that it is used without complaints in other places, that there is config somewhere to suppress the error?